db/migrate/20181206142708_create_other_asset_declaration.rb

```diff
@@ -8,8 +8,6 @@ def change
       t.decimal :timeshare_value
       t.decimal :land_value
       t.decimal :jewellery_value
-      t.decimal :vehicle_value
-      t.decimal :classic_car_value
```

Comment:
You should never modify previous migrations. It is not needed and it will mess things up.
Running `db:migrate` on a fresh database would fail because the migration removing these columns would try to remove columns that were never added in the first place.

Reply:
Ah ok thanks. Must have happened during a rebase issue I once had.